### DISCUSSION TITLE
Redis Adopts Dual Source-Available Licensing

### DISCUSSION THREAD

**Discussion by Abhi5h3k** [StackStorm/st2#6248] [Q&A]
Will the change in Redis License Affect Stackstorm redis container! [redis-adopts-dual-source-available-licensing](https://redis.io/blog/redis-adopts-dual-source-available-licensing/)

Redis has changed their license away from 3-clause BSD. [redis/pull/13157](https://github.com/redis/redis/pull/13157)

**Comment by nzlosh** [comment]
Duplicate https://github.com/StackStorm/st2/discussions/6172
